### DISCUSSION TITLE
Add "Flash write Count" to INFO3 JSON

### DISCUSSION THREAD

**Discussion by ortegafernando** [arendst/Tasmota#14785] [Q&A] [answered]
HI, related with https://github.com/arendst/Tasmota/discussions/14747

To detect abnormal operation it would be ideal to have this also: FLASH WRITE COUNT in INFO3 JSON.

Thanks

**Comment by sfromis** [comment, marked as answer]
Instead of asking for more and more fields, you can just create a very simple rule to do a status command at boot time.

**Reply by ortegafernando** [reply]
I understand your option, but I think that it doesnt involves any overload , over-sketchsize, .... This information is very important so it is better to have it in all tasmota (inside sketch) and not to configure a rule in all tasmotas.

I understand that you and me have different points of view of the solution, but as this change is very very little and has very very little impact in sketch size, I think is a good way to warn everybody: "ey!, be carefull, control this data in INFO3, just in case...."

But I must admit that your solution is ok, but I don't like it as much as mine  :wink:

**Reply by ortegafernando** [reply]
PR created: https://github.com/arendst/Tasmota/pull/14822
